mindmeld/models/embedder_models.py

```diff
@@ -793,6 +901,12 @@ def _encode(self, text_list):
                 encoded_vecs.append(np.average(vl, axis=0))
         return encoded_vecs
 
+    def _tokenize(self, text):
+        return [
+            t["entity"] for t in
+            self.text_preparation_pipeline.tokenize_and_normalize(text)
+        ]
+
     def dump(self):
```

Comment:
So embedder models' dump functionality is just dumping and loading a cache object;, and no configs are dumped. Because all embedder classes have the cache in the same way, I guess we don't need an embedder specific dump? @mhuang2173 

And for the overlapping of dump for Glove, I would like to take that up in PR 325 with some more modifications to the Glove embedder class.